### PR TITLE
fix LoadAnnotations representation wrong message

### PR DESCRIPTION
**Pull request by hyq12358**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Modification

modify representation info of pipeline.LoadAnnotations

### CODE REVIEW

**Review by RangiLyu — APPROVED** [review]
LGTM

File: mmdet/datasets/pipelines/formating.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def __call__(self, results):
                 continue
             results[key] = DC(to_tensor(results[key]))
         if 'gt_masks' in results:
-            results['gt_masks'] = DC(results['gt_masks'], cpu_only=True)
+            results['gt_masks'] = DC(to_tensor(results['gt_masks']), cpu_only=True)
```
Comment (RangiLyu) [inline mmdet/datasets/pipelines/formating.py:216]: Please do not submit a commit that is not part of this PR.

Comment (hhaAndroid) [reply mmdet/datasets/pipelines/formating.py:216]: Can not be modified.

**Comment by ZwwWayne** [comment]
Cannot be merged as CLA is not signed. Postpone it to the next version.

**Comment by OpenMMLab-Assistant005** [comment]
Hi @hyq12358 ！We are grateful for your efforts in helping improve this open-source project during your personal time. 
Welcome to join OpenMMLab Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA
If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤
